Add marketing content

thoiberg opened the pull request:
To upload a plugin to the Magento store the plugin needs to pass both a [technical review](https://devdocs.magento.com/marketplace/sellers/submit-for-technical-review.html) and a [marketing review](https://devdocs.magento.com/marketplace/sellers/submit-for-marketing-review.html). The marketing review requires the following items to be submitted:

- A Product Icon
- A short description
- A long description
- A thumbnail
- "at least two high-quality images for the image gallery that depict the extension"
- one of the following: user guide, installation guide, or reference manual (must be in PDF format)

Since this content is so closely related to the plugin I decided to include it with the code in the repo.

For the detailed description I'm planning to convert [the README](https://github.com/komoju/komoju-magento/blob/master/README.md) into a PDF (and lightly edit it), but I'm currently waiting to find out what the plugin link will be so we can include that in the installation instructions before submitting (which I believe I will get during the technical submission)

thoiberg committed:
Add Marketing directory to the docs

When submitting the extension to Magento for listing on their
marketplace, we need to pass a marketing review as well as a technical
review. To ensure that anyone working on this project in the future has
all the assets they need all marketing material will be included in a
docs/ folder in the repository.

thoiberg committed:
Add short description

Adding the short description content. This field can only be 150
characters/

thoiberg committed:
Add screenshots

Since the plugin integration is pretty sparse I just grabbed one plugin
of checking out with Komoju and one of configuring the plugin.

thoiberg committed:
WIP: add detailed description

thoiberg committed:
WIP: more updates for detailed description

thoiberg committed:
Complete detailed description

thoiberg committed:
Update screenshots

We've made some style and branding tweaks to the extension, so updating
the screenshots to ensure they match the currect wording.

thoiberg committed:
Add extension icon

Using the KOMOJU logo text as the extension icon as that seems like the
most appropriate asset to use that I could find. Magento requires the
icon to be 240x240px so rather than horribly distorting the text I kept
it at 240px width and filled in the rest with a white background.

thoiberg committed:
Add screenshots to README

Improving on the README so we can use it as an installation guide for
the marketing review. Some additional screenshots have been added to the
setup process to make it easier to follow along.

thoiberg committed:
Add note on README being install guide

thoiberg commented on docs/marketing/short-description.md:
```diff
@@ -0,0 +1 @@
+Built for developers, KOMOJU is the easiest way to integrate Japanese payments into your Magento store.
```
We have 150 characters max here.

thoiberg committed:
Change marketing instances of Komju to KOMOJU

jdgc commented on README.md:
```diff
@@ -37,11 +37,22 @@ $ php bin/magento setup:static-content:deploy
-To ensure that the Magento plugin works correctly you will need to set up a webhook from your Komoju dashboard to the wordpress instance. To do this you will need to go to your [Webhook page on the Komoju dashboard](https://komoju.com/admin/webhooks) and click "New Webhook". The Webhook URL is at `/komoju/hostedpage/webhook` of your Magento website. If your Magento was `https://magento.komoju.com ...
+When configuring the "API Settings" section of the plugin use the following values:
+
+"Komoju merchant ID" in the plugin is "Merchant UUID" in the Merchant Settings dashboard.
+"Secret Key from Komoju" in the plugin is "Secret Key" in the Merchant Settings dashboard.
```
We should capitalize these here and also in the plugin source

thoiberg replied:
Yep good thinking. Honestly, I think I'll just change the fields in the plugin to be the exact same name as the fields in the Merchant Settings page, that way we can eliminate any confusion.

thoiberg replied:
Updated in ed659bd

jdgc commented on docs/marketing/detailed-description.md:
```diff
@@ -0,0 +1,18 @@
+
+## Features
+
+- Allow your customers to pay with the methods you prefer best: Credit Card or Konbini
```
Can we modify this description after release if we decide to add more payment methods?

thoiberg replied:
Yep! It will require us to submit it for review again, but since we would have to make changes to the code to support additional payment types we would have to get it re-reviewed anyway.

[Source](https://devdocs.magento.com/marketplace/sellers/extension-update-information.html#update-marketing-information)

jdgc commented on docs/marketing/detailed-description.md:
```diff
@@ -0,0 +1,18 @@
+## Security
+
+- PCI Compliant
+- No Credit Card details stored in Magento
```
I think we can leave out the second point since its implied by the PCI compliance. Maybe we can remove this section and add something about PCI compliance under features

thoiberg replied:
Done. I've rolled it into ["Features"](https://github.com/komoju/komoju-magento/pull/4/commits/8ebe88d740fbd06bc9c8c1e64d55c506d41e492c):
```
PCI Compliant; We do the hard work of keeping your user's data secure so you don't need to worry.
```

jdgc reviewed (approved):
Images look good, text is broadly good. Have some nitpicks that you can take or leave. Thanks!

thoiberg committed:
Change API setting field names

Changing the API setting field names to mirror the fields on the
KOMOJU Merchant Settings dashboard. Thi way any users setting up the
plugin with their Magento store don't need to try and figure out which
values they should be using. Updating the screenshot to reflect the new
wording.

thoiberg committed:
Update marketing copy based on feedback

thoiberg committed:
Update dictionary files with new wording

thoiberg committed:
Fix apostrophe position

Hopefully you would have multiple users and not just the one. English is
confusing :(.
